playbooks/roles/openvpn/templates/client-direct-udp.ovpn.j2

```diff
@@ -0,0 +1,3 @@
+client
+remote {{ openvpn_server }} {{ openvpn_port_udp }}
+{% include "client-common-udp.j2" %}
```

Comment:
Just in case the other comment wasn't clear, I'm suggesting that if we had the `proto` line in this file (and the other OVPN templates) then we could include `client-common.j2` here and wouldn't have to duplicate a bunch of the same lines in a separate UDP-only file.